dotcom-rendering/src/components/marketing/banners/common/types.tsx

```diff
@@ -80,4 +80,5 @@ export interface BannerRenderProps {
 	tracking: Tracking;
 	submitComponentEvent?: (componentEvent: ComponentEvent) => void;
 	design?: ConfigurableDesign;
+	promoCodes?: string[];
```

Comment:
we now have to pass this field through to the banner component for use by `enrichSupportUrl`